push_notifications/conf/app.py

```diff
@@ -0,0 +1,357 @@
+from django.core.exceptions import ImproperlyConfigured
+from django.utils.six import string_types
+from .base import BaseConfig, check_apns_certificate
+from ..settings import PUSH_NOTIFICATIONS_SETTINGS as SETTINGS
+
+"""
```

Comment:
I sometimes see a pattern of shipping an example project, [like in allauth](https://github.com/pennersr/django-allauth). Don't sweat it though.